data/multiplayer/scenarios/ANL_utils/ANL_leader_options.cfg

```diff
@@ -369,7 +374,7 @@ Help an ally with their research")}
                                         [message]
                                             caption= _ "Diplomatic Options"
                                             speaker=unit
-                                            image=items/book4.png
+                                            image= "items/book4.png"
```

Comment:
I would, but it _is_ just a single line, so it's not hugely important. More important are all the whitespace problems.